docs/meetings/client/2022_10_07.md

```diff
@@ -0,0 +1,119 @@
+    - Generates an MP4, a silhouette, and then creates the silhouette of a face moving, and then anchors the face to the body
+    - Can take time and can struggle with larger images, appears robotic
+- 3D
+    - Already built heads and faces
+    - Input from audio, uses libraries and pre-trained models to animate the speaker
+    - Found iClone Aids? Commercial and comes at a high cost, but very in-depth and looks very good
+    - Found Viseme generation; licensed for non-commercial or research work. More research needed for realism and a good-looking product
+- Nazim Questions
+    - Work on client side, in browser, through Google Collab
+    - Erik, what is preferable?
+        - Service-side rather than client-side; API would be better
+        - Back-end runs on Amazon EC2
+
+## TEAM C
+
+- Research on 3 methods
+
+1.	Pre-trained model (Wav2Lip) upload an image or video, audio as well, and generate an output video, roughly 2-min for a 1-min video (scaling). Commercial use has contact info, but no pricing information. No understanding of the limit of length of audio or video.
+
+2. NVIDIA (Audio to face) neural net takes audio source and can sync it to any 3D object. Probably supported for a while, but it will require licensing fees that are not clear. Probably resource-heavy because it requires processing power. Result looks really good. Processing time will probably be heavy as well
+
+3. Lazy KH approach (repo name), maps audio to phonemes and mouth expression. Doesn’t use any kind of deep learning or machine learning. Least computationally expensive. Not uploading an image, so it’s difficult to upload an avatar, but you can have some control (upload a base avatar). User can upload different images and textures, face avatar and mouths and eyes, and then can upload audio and text, and the approach will animate the mouth speaking on the avatar. More difficult with an upload. Very simple to implement, very little processing power, only available on Mac, from 2015, and it’s slow – one-off project but they could optimize it if they take it and customize it
+
+- Nazim: Timing? Time to compute audio to face animations is very quick, but the problem is you need high-end hardware with good computing power. Resources would be quite high.
+- Erik: Micro or medium resources currently, nothing insane. The NVIDIA solution might be too heavy. Can always increase computing power or isolate it with an API, Kukarella would need to weigh the options.
+ 
+# (16:45) QUESTIONS FOR KUKERELLAS TECH
+
+- Erik:
+    - would like a small written-up report to review in more detail
+    - ach team could try a different tech stack
+
+- Nazim:
+    - Some research solutions can come from unexpected places
+    - Let’s try different approaches
+
+# Student Questions
+
+- Audio file format: MP3 and WAV files produced
+- Video file format preferred: MOV and MP4
+
+Conclusion:
+-	TEAM A: We prefer LipGAN
```

Comment:
We decided we no longer prefer LipGAN - not sure if this is just because of what we outlined in the actual meeting, but might be worth changing to the actual approach we have decided on